test/e2e/clusterctl_upgrade.go

```diff
@@ -467,6 +470,10 @@ func ClusterctlUpgradeSpec(ctx context.Context, inputGetter func() ClusterctlUpg
 			testCancelWatches()
 		}
 
+		if input.PreCleanupManagementCluster != nil {
+			By("Running Pre-CleanManagement steps against the management cluster")
```

Comment:
nit: let's adjust the message accordingly

Reply:
Oops nice catch I missed that